### ISSUE TITLE
Making <when> element refer to arbitrary points in the score

### ISSUE THREAD

**Issue by lpugin** [music-encoding/music-encoding#574]
In `<when>` elements (for example within ./music/performance/recording/clip) it is possible to refer to elements such as `<note>` with `@data`. However, a way to refer to a point where no musical event occurs is currently lacking. It would be useful to have a way to refer to a point in a timestamp manner.

**Comment by kepper** [comment]
What you expect to have absolute timestamps, that is, from the beginning of the work (or more precisely, `<mdiv>`), or would you want to refer to the measure first, and then specify the timestamp in relation to that measure (leaving aside older repertoires for now…)?

**Comment by lpugin** [comment]
A simple way would indeed to refer to a measure first and then to be able to add a timestamp value. Referring from the beginning could be possible, but then timestamp would need to have the form of `tstamp2`, and I don't think this would be ideal in all cases.

**Comment by kepper** [comment]
Would it be sufficient to allow tstamp-based values for `@interval` (plus `@inttype`, which ultimately means within https://music-encoding.org/guidelines/v4/data-types/data.betype.html), in conjunction with `@since` referring to the measure?

**Comment by lpugin** [comment]
Yes, I think this would work. The definition of `when@since` also needs to be revised because now it explicitly requires the reference to be another `<when>` element.

**Comment by kepper** [comment]
We need to step back a little. First I though that it would be sufficient to have a `<when>` for every measure, and then just relate to that. However, we're expressing that "relative" time in two ways here: Both as actual time (milliseconds in the audio), and as musical time (timestamps within the measure). Since `@since` is intended to capture the first, it can't to the second at the same time. Looks like we need new attributes :-/
